versions.yaml

```diff
@@ -227,9 +227,9 @@ externals:
   coco-trustee:
     description: "Provides attestation and secret delivery components"
     url: "https://github.com/confidential-containers/trustee"
-    version: "18c8ee378c6d83446ee635a702d5dee389028d8f"
+    version: "6adb8383309cbb7279f1d8e1e4620556ac66481e"
```

Comment:
There are newer hashes available. Any reason to choose this one in particular?

Reply:
@fitzthum, I've decided to follow the rule of "bumping to use what we need", as we needed the kustomization for TDX.